### ISSUE TITLE
Add option to ignore box-shadows

### ISSUE THREAD

**Issue by benjamincharity** [ierhyna/stylelint-no-indistinguishable-colors#5]
Somewhat similar to #4, this plugin (which is great btw) is throwing a warning for the following code:

```scss
box-shadow: 0 3px 1px -2px rgba(0, 0, 0, .2), 0 2px 2px 0 rgba(0, 0, 0, .14), 0 1px 5px 0 rgba(0, 0, 0, .12);
```

While I do want this warning when using those colors normally, in the case of custom shadows I think it's fairly common to have various shades that are quite similar.

It would be great to have an option to ignore box-shadows. (I could always move this code to a variable if #4 is solved first)
